ops-devnet/devnet-up.sh

```diff
@@ -105,6 +125,8 @@ VALPOOL_ADDRESS="0x6900000000000000000000000000000000000005"
   L2OO_ADDRESS="$L2OO_ADDRESS" \
     COLOSSEUM_ADDRESS="$COLOSSEUM_ADDRESS" \
     VALPOOL_ADDRESS="$VALPOOL_ADDRESS" \
+    VALIDATOR_MNEMONIC="$VALIDATOR_MNEMONIC" \
+    VALIDATOR_HD_PATH="$VALIDATOR_HD_PATH" \
```

Comment:
Is there any way to avoid setting these values as environment variables?

Reply:
Maybe via file input? Do you have any ideas?

Reply:
I agree that it'd be better to change not to use env var for mnemonic, but for that it seems that we need to change the way of config. The mnemonic (or private key) of batcher is also passed via env var.

Reply:
Removed these lines by changing to run the `deposit` command after bringing up the validator node.